index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<link rel="stylesheet" type="text/css" href="css/styles.css">
+  <title>Startup Matchmaker</title>
+</head>
+<body>
+  <header>
+    <div class="page-header">
+      <div class="content">
+        <h1>Startup Matchmaker</h1>
+        <nav class="page-nav">
+          <a class="" href="">Find Developers</a>
+          <a class="" href="">Find Designers</a>
+          <a href="">How it Works</a>
+          <a href="">Our Team</a>
+          <a href="">Blog</a>
+        </nav>
+      </div>
+    </div>
+    <div class="page-subheader">
+      <div class="content">
+        <h2>Because two brains <span>tastes</span> <u>are</u> better than one.</h2>
```

Comment:
Try not tot use decorative tags to style your content, please apply your styles from your external stylesheet.